.github/workflows/build-all.yml

```diff
@@ -0,0 +1,114 @@
+name: Build Bowtie Image for all available versions
+
+on:
+  workflow_dispatch:
+
+env:
+  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  versions:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    outputs:
+      revisions: ${{ steps.revisions.outputs.value }}
+    steps:
+      - name: Collect available versions
+        id: revisions
+        run: |
+          tags=$(gh api \
```

Comment:
Do we really need the GH API? Shouldn't this be doable just using the git repo, just checkout the whole history rather than just shallow?

Reply:
Yes, we can use `git` only. But GH API returns more structured output, which makes it easier to post-process. If you have some objections about using GH API here I will migrate to using `git` only

Reply:
I expected as much, yeah :)